src/generic-container/generic-container.test.ts

```diff
@@ -380,6 +380,22 @@ describe("GenericContainer", () => {
     expect(await getRunningContainerNames()).not.toContain(containerName);
   });
 
+  it("should handle health checks without the need of a shell", async () => {
```

Comment:
i think the new test should be based off `should wait for custom health check`, with the use of `CMD` being called out very clearly

Reply:
👍